### DISCUSSION TITLE
got no permission

### DISCUSSION THREAD

**Discussion by natank1** [community/community#137837] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am wiating three weeks for permsission for the Beta version , How come it is that long?

**Comment by queenofcorgis** [comment]
Hi @natank1 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have closed the discussion, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
